tools/cfn2ts/lib/codegen.ts

```diff
@@ -500,9 +504,10 @@ export default class CodeGenerator {
         },
       });
 
-      self.code.line(`  ${cfnName}: ${mapperExpression}(properties.${propName}),`);
+      self.code.line(`${cfnName}: ${mapperExpression}(properties.${propName}),`);
     });
-    this.code.line('};');
+    this.retainPropertiesNotInCfnSchema(Object.values(nameConversionTable), 'properties');
```

Comment:
The way this function is named confuses me. The `verb` "retain" makes it look like this has a side effect in some data structure, but it actually means "RENDER the properties that aren't in the CFN schema here", right?

Maybe something like `renderUnmodeledProperties` or something? With a comment line explaining why:

```
// These properties were in the original template but can't be represented in the L1 model,
// so add them back in here.
```

Or something.

Reply:
Done! Renamed to `renderSavingPropertiesNotInCfnSchema()`.